.github/workflows/flake8.yaml

```diff
@@ -0,0 +1,16 @@
+name: Flake8
+
+# Trigger the workflow on push
+on:
+  push:
```

Comment:
If you don't pin the flake8 version, you should have a scheduled (weekly?) run.

Reply:
Fixed.